submissions/golden_handcuffs.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Compensation Manipulation"
+date: 2024-04-04T21:08:37-07:00
+draft: false
+keywords:
+  - FAANG
+  - golden handcuffs
+  - big tech
+  - compensation
+  - tech jobs
+tags:
+  - FAANG
```

Comment:
```suggestion
  - FAANG
  - big tech
  - compensation
  - tech jobs
  - thoughts
```

adding these to tags will enable quick navigation to other similarly tagged future posts